### ISSUE TITLE
PyTorch build fails for Visual Studio 2022

### ISSUE THREAD

**Issue by mworchel** [NVlabs/nvdiffrast#81]
First of all, thanks for the awesome work!

I noticed that building the PyTorch extension fails for Visual Studio 2022 with
> "Could not locate a supported Microsoft Visual C++ installation"

It seems like the default install path for Visual Studio has been changed from `C:\Program Files (x86)` to `C:\Program Files`, which results in `cl.exe` not being properly detected.

Since you don't merge pull requests in the GitHub repository, I propose a solution here:
Replace this line (https://github.com/NVlabs/nvdiffrast/blob/main/nvdiffrast/torch/ops.py#L32): 
```python
...
paths = sorted(glob.glob(r"C:\Program Files (x86)\Microsoft Visual Studio\*\%s\VC\Tools\MSVC\*\bin\Hostx64\x64" % edition), reverse=True)
...
```
with
```python
...
vs_relative_path = r"\Microsoft Visual Studio\*\%s\VC\Tools\MSVC\*\bin\Hostx64\x64" % edition
paths = sorted(glob.glob(r"C:\Program Files" + vs_relative_path), reverse=True) + sorted(glob.glob(r"C:\Program Files (x86)" + vs_relative_path), reverse=True)
...
```

A similar fix should be applied to the TensorFlow bindings (https://github.com/NVlabs/nvdiffrast/blob/main/nvdiffrast/tensorflow/plugin_loader.py#L58-L74).

Thanks!

**Comment by s-laine** [comment]
Thank you for the report and the suggested solution! I'll fix this in the next release -- keeping this issue open until then.

**Comment by s-laine** [comment]
Fix included in v0.3.0, closing. Thanks again!
